day-1/inventories/prod/group_vars/all/vars.yaml

```diff
@@ -3,6 +3,29 @@ ansible_ssh_common_args: "-F ./hcloud_ssh_config_rpch_production"
 
 environment_name: "prod"
 
-hoprd_version: providence
+hoprd_version: 2.0.6
 hoprd_network: dufour
 rpch_exit_node_version: 0.14.1
```

Comment:
can you rebase to master?

this one is 1.1.1

Reply:
Done